packages/protocol/package.json

```diff
@@ -10,12 +10,11 @@
     "lint:sol": "pnpm prettier '**/*.sol' --write && pnpm solhint 'contracts/**/*.sol' --fix",
     "eslint": "pnpm exec eslint --ignore-path .eslintignore --ext .js,.ts .",
     "eslint:fix": "pnpm exec eslint --ignore-path .eslintignore --ext .js,.ts . --fix",
-    "test": "pnpm hardhat test --grep '^[^integration]'",
-    "coverage": "pnpm hardhat coverage --solcoverjs ./.solcover.js --show-stack-traces",
-    "test:coverage": "pnpm test && pnpm coverage",
+    "test:unit": "pnpm hardhat coverage --solcoverjs ./.solcover.js --show-stack-traces --testfiles 'test/unit/bridge/*.ts test/unit/bridge/**/*.test.ts test/unit/common/*.test.ts test/unit/L1/*.test.ts test/unit/L2/*.test.ts test/unit/libs/*.test.ts test/unit/thirdparty/*.test.ts'",
+    "test:protocol": "DOTENV_CONFIG_PATH=.env.protocol pnpm hardhat coverage --solcoverjs ./.solcover.js --show-stack-traces --testfiles 'test/unit/protocol/*.test.ts'",
```

Comment:
why do we need these separate? can we combine them? it feels odd to have unit/L1 and unit/L2 folder, and also a unit/protocol folder with TaikoL1.test file

Reply:
Let me reconsider the whole PR. Marking it as draft for now.